css/homepage.css

```diff
@@ -115,10 +115,6 @@ nav {
   justify-content: center;
 }
 
-.icons:hover {
-  background: #c5e9da;
-}
-
 @media screen and (min-width: 768px) {
   .hamburger,
   .header-title {
```

Comment:
- [ ] The background image appears to be covering some the text as well as the nav menu links as shown below.
 Kindly check and ensure it matches the actual design. :wink: 

### Your design 

![Screenshot from 2023-02-25 12-04-45](https://user-images.githubusercontent.com/65233860/221356285-8d8022f0-d68b-4275-9092-cabe475150a8.png)


### Figma design

![Screenshot from 2023-02-25 12-06-44](https://user-images.githubusercontent.com/65233860/221356292-c127123a-a129-4d74-bcf9-f43508f12aa4.png)